modules/react/button/stories/button/Button.stories.mdx

```diff
@@ -15,6 +17,9 @@ import {SecondaryInverse} from './examples/SecondaryInverse';
 import {Tertiary} from './examples/Tertiary';
 import {TertiaryInverse} from './examples/TertiaryInverse';
 import {Delete} from './examples/Delete';
+import {Link} from './examples/Hyperlink';
+import {LinkInverse} from './examples/HyperlinkInverse';
+import {LinkExternal} from './examples/HyperlinkExternal';
```

Comment:
This is a small nit, and it really has to do with the overall organization of stories. Because these examples are called Link, Link Inverse, and Link External, those names show up in the left nav and searching for "hyperlink" removes them from the results, which is some nonsense outside of our control.

![searching the storybook left nav for hyperlink and seeing the aberrant behavior described above](http://g.recordit.co/T19iCZAI8J.gif)

Reply:
But that kind of makes me think these examples should live elsewhere.

```
Components
|- Hyperlink
|  |- Basic
|  |- External
|  |- Inverse
```

Reply:
100% agree

Reply:
@alanbsmith should we save reorganizing to a new component for v7?